### DISCUSSION TITLE
Security Onion 2.3 enabling Kibana reporting

### DISCUSSION THREAD

**Discussion by davidcerve7** [Security-Onion-Solutions/securityonion#4269] [Unsupported Versions] [answered]
Hello All, 

Complete noob here, so please bear with me. 

I recently set up a security onion server to receive Syslog messages and can confirm it works. I would like to export some of the logs in CSV format, which I know can be done using the reporting feature in Kibana. I understand that I need to modify the kibana.yml to enable this feature but Security Onion 2 uses Salt and pillars to manage the configuration files. When I modify the. yml under /opt/so/conf/kibana. Kibana breaks and the configuration gets overwritten back to the original. 

From the documentation, it looks like Security Onion uses salt and pillars to manage some configuration files. I am, unfortunately not experienced using salt and was hoping someone can guide me in the right direction on where I could make these changes. I appreciate the help!

Security Onion version:  2.3.3
Standalone Installation

**Comment by weslambert** [comment, marked as answer]
2.3.50 should have this feature available by default IIRC. 

Given that it should be enabled, you could then do something like the following:

https://github.com/Security-Onion-Solutions/securityonion/discussions/1902#discussioncomment-128679

(Disregard the portion about `so-features-enable`)

**Reply by davidcerve7** [reply]
@weslambert , I updated to the latest version and that enabled it. Thank you!
